spec/classes/agent_spec.rb

```diff
@@ -157,6 +157,9 @@
         let(:pre_condition) do
           "class { 'sensu': manage_repo => false }"
         end
+        if facts[:os]['family'] != 'windows'
```

Comment:
please add a comment as to why windows is not tested

Reply:
Comment added